carsus/io/output.py

```diff
@@ -0,0 +1,25 @@
+from pandas import read_sql_query
+from carsus.model import Atom, AtomWeight, DataSource
+
+
+def read_basic_atom_data(session):
```

Comment:
Like the `AtomData` class from [tardis/atomic.py](https://github.com/tardis-sn/tardis/blob/master/tardis/atomic.py)? It can have attributes that are required dataframes and use functions like `read_basic_atom_data` (although with different name) to create these dataframes:
```
self.basic_atom_data = create_basic_atom_data(session)
```